### ISSUE TITLE
Enemy teleporting when pulling the corpse out of a wreck with a harpoon gun

### ISSUE THREAD

**Issue by JHMota** [Regalis11/Barotrauma#7309]
**Description**
In the attached video you can see me killing Leucocytes with a harpoon gun and whenever one is pulled past the border of the wreck and open sea they teleport somewhere with my harpoon. This happens at the top of wreck too where I was testing light glitching when passing the "border". After teleporting Leucocytes for a while I noticed a significant fps drop.
This could be linked to players teleporting from ruins which has happened to me only once so far but that also happened when I exited ruins to open sea.

Edit: I only have one mod on (GunLights Remade) but I doubt that has anything to do with it.
Edit2: I just got teleported from ruins again. I had those small red creatures (Swarm Feeders?) attached to my characters body when I exited ruins. So it seems when any enemy is "forced" to exits the prefab they spawned in, it will teleport anything they are attached to and then disappear.

https://user-images.githubusercontent.com/36190400/141687482-831badf1-306d-479a-a77c-4ea8463e17b8.mp4

**Steps To Reproduce**
Shoot an enemy that is inside a wreck with harpoon gun and let the harpoon gun pull them out of the wreck.

**Version**
Version 0.15.13.0 (ReleaseWindows, branch release, revision 4e66dd7b5d)

Hope this helps and keep up the good work.
-Mota

**Comment by Regalis11** [comment]
Duplicate of https://github.com/Regalis11/Barotrauma/issues/6922
